Makefile

```
@@ -511,6 +512,14 @@ endif
 endif
 endif
 
+ifeq ($(ENABLE_GHDL),1)
+GHDL_DIR ?= /usr/local
```

Should this not be `$(PREFIX)`? E.g.:

```suggestion
GHDL_PREFIX ?= $(PREFIX)
```

---

Indeed...